service_container.rst

```diff
@@ -923,6 +923,34 @@ them will not cause the container to be rebuilt.
     means that all classes are "available to be *used* as services" without needing
     to be manually configured.
 
+The namespace Option
```

Comment:
What about enclosing the following section in a `sidebar` directive instead? IMO it's a nice addition for the YAML format, but not a totally new concept.

Reply:
Done.